pyproject.toml

```diff
@@ -6,18 +6,28 @@
 # verbose = true
 line-length = 80
 skip-string-normalization = true
+
 [tool.isort]
 profile = "black"
 known_first_party = "kobo"
 no_lines_before = ["LOCALFOLDER"]
 
 [tool.ruff]
-line-length = 80
+line-length = 88
```

Comment:
Why are we making the line-length longer for ruff than for black?

Reply:
Because we use an extra buffer of ~10 characters. If your line is 85 characters and you don't want to break it for 5 characters, we don't want the linter to complain about it. Moreover, that's the default configuration (see https://docs.astral.sh/ruff/configuration/)

> # Same as Black.
> line-length = 88
> indent-width = 4

I would leave black as-is to format with 80 characters. Does not mean the dev cannot bring a line up to 88 characters after Black formatting.

Reply:
In PyCharm, I added two visual limits. One at 80 (kinda soft-limit) and the other one at 90 (kind hard-limit).

![Screenshot 2024-08-28 at 09 36 09](https://github.com/user-attachments/assets/4c4ae594-edfc-4d34-9499-4572f07df4d7)

Reply:
I'm still a little confused. Why do we have black at 80 characters then?

Reply:
Point still holds. Why do we cap black at 80?